components/Tabs/Tabs.js

```diff
@@ -87,6 +79,5 @@ class TabCard {
 let tabs = document.querySelectorAll('.tab');
 
 tabs.forEach(tab => {
-  //console.log(tab);
   new TabLink(tab);
```

Comment:
Haja You did great job on the sprint challenge. You properly implemented Tabs and Cards components into the project and were able to filter out articles by selecting particular Tab. 
Overall:
* Tabs and Cards properly implemented (MVP)
* Great answers to questions (MVP)
* Clean Code
* Fluid git workflow

- For possibile improvements try and add more articles to the project 
- Great job, You displayed good understanding on the subject 👍